src/components/proposalActions.jsx

```diff
@@ -255,17 +239,17 @@ const ProposalVote = ({
                         mt='-4px'
                       />
                     </Tooltip>
-                  ) : null}
+                  ) : null} */}
```

Comment:
Is this supposed to be commented out? if yes, maybe worth removing

Reply:
Should probably put that back in, lol.